envoy.dependency.check/envoy/dependency/check/abstract/dependency.py

```diff
@@ -115,7 +115,8 @@ async def newer_release(
             self.release_class(
                 self.repo,
                 newer_release.tag_name,
-                release=newer_release)
+                release=newer_release,
+                asset_url=self.github_url)
```

Comment:
i dont think this is needed as the newer_release version of release doesnt check shas